### DISCUSSION TITLE
Philips dicom headers

### DISCUSSION THREAD

**Discussion by MariaLiljeroth** [GSTT-CSC/hazen#393] [General]
Hi, has anyone had any issues with Philips (classic) dicom headers ?

![image](https://github.com/GSTT-CSC/hazen/assets/33836457/1e29ca59-1252-4b00-a94c-2a98a56b428d)

**Comment by sophie22** [comment]
The input to hazen tasks if a directory path (folder) which is expected to contain exactly 11 images corresponding to the slices of the ACR phantom. Please refer to the ACR guidance on the specifics of how to acquire the images.

Based on the screenshot provided it looks like your input is a single file (`00001`) rather than a folder.
Please try running the following command instead: `hazen acr_snr .\` and let me know if that still causes errors.
